src/javascript/gameInstance.js

```diff
@@ -383,7 +383,7 @@ export default class GameInstance {
     }
   }
 
-  getCurrentTime() {
+  async getCurrentTime() {
```

Comment:
I *think* you forgot to make `this.audio.getCurrentTime()` into an immediately resolved `Promise`.

```js
Promise.resolve(this.audio.getCurrentTime());
```

Because otherwise `this.ytPlayer.getPlayerTime()` returns a `Promise` but `this.audio.getCurrentTime()` does not...

Reply:
Sorry, my bad lol